test/rosdep_repo_check/config.yaml

```diff
@@ -66,8 +66,8 @@ supported_versions:
   debian:
   - bookworm
   fedora:
-  - '40'
   - '41'
+  - '42'
```

Comment:
CI is validating that all Fedora rules are valid (i.e., point to valid packages) when a new version is added, right? I'm thinking of rules that specify `'*'` instead of `41`.

Reply:
CI only validates rules as they are added.

There are likely a small number of rules which are valid for Fedora 40 and/or 41 which are not valid for Fedora 42. As long as those rules are not modified by a PR, they won't be checked and won't regress the checks.